application/src/main/java/org/thingsboard/server/service/edge/EdgeEventSourcingListener.java

```diff
@@ -112,7 +112,7 @@ public void handleEvent(DeleteEntityEvent<?> event) {
             return;
         }
         try {
-            if (EntityType.TENANT.equals(entityType) || EntityType.EDGE.equals(entityType)) {
+            if (EntityType.TENANT == entityType || EntityType.EDGE == entityType || EntityType.AI_MODEL_SETTINGS == entityType) {
```

Comment:
I guess this entity should be synchronized to Edge same as all other entities. Please ask Yevhenii Zahrebelnyi to add support for that